docs/source/getting_started/index.md

```diff
@@ -62,7 +62,7 @@ llama-stack-client --endpoint http://localhost:$LLAMA_STACK_PORT models list
 You can test basic Llama inference completion using the CLI too.
 ```bash
 llama-stack-client --endpoint http://localhost:$LLAMA_STACK_PORT \
-  inference chat_completion \
+  inference chat-completion \
```

Comment:
![image](https://github.com/user-attachments/assets/527e706d-b0cd-4691-ae96-82787ea2be45)

Quick start guide: https://llama-stack.readthedocs.io/en/latest/getting_started/index.html

chat-completion not chat_completion:

https://github.com/meta-llama/llama-stack-client-python/blob/eb3d21932d22b863e5687eed2ce26e713ab46682/src/llama_stack_client/lib/cli/inference/inference.py#L21